packages/pages/src/vite-plugin/build/build.ts

```diff
@@ -46,6 +46,19 @@ export const build = (projectStructure: ProjectStructure): Plugin => {
           },
         },
         define: processEnvVariables(projectStructure.envVarPrefix),
+        experimental: {
```

Comment:
Vite recommends pinning to a minor version when using experimental builds https://vitejs.dev/guide/build.html#advanced-base-options

Reply:
Talked offline, but this won't change until the next major - https://github.com/vitejs/vite/pull/8762#issuecomment-1527147910